osmosis-1/osmosis-1.assetlist.json

```diff
@@ -919,6 +919,34 @@
         "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/dsm.png",
         "svg": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/dsm.svg"
       }
+    },
+    {
+      "description": "The native staking and governance token of the Osmosis.",
+      "denom_units": [
+        {
+          "denom": "ibc/307E5C96C8F60D1CBEE269A9A86C0834E1DB06F2B3788AE4F716EDB97A48B97D",
+          "exponent": 0,
+          "aliases": [
+            "udig"
+          ]
+        },
+        {
+          "denom": "dig",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/307E5C96C8F60D1CBEE269A9A86C0834E1DB06F2B3788AE4F716EDB97A48B97D",
+      "name": "Dig Chain",
+      "display": "dig",
+      "symbol": "DIG",
+      "ibc": {
+        "source_channel": "channel-1",
+        "dst_channel": "channel-128",
+        "source_denom": "udig"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/DIG.png"
```

Comment:
https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/dig.png

Reply:
I fixed it. Thanks for your help !!!